### PR TITLE
style: change popups style by a designer's guidance

### LINKED ISSUE #495: [Feature] More detail should be updated for hover popup panels.
### Description

After the great feature in #395 , we got several new panels on GitHub. But I think we need more details refined for the feature, the problems I encounter are:

- The new panels do not support i18n, they are always in English, so we should make them support i18n and follow the config.
- The new panels do not support light mode and dark mode change, so we should make two different appearances for different themes.
- The panels are usually used as screen snapshots to share. But right now the panels have a transparent backgrounds, so the screenshots may contain other elements on the page. So maybe we should remove the transparency.
- Using line chart and bar chart in a same panel looks a little bit strange to me. I think maybe make the line chart more smooth may make it better.

### PR DESCRIPTION
**Pull request by tyn1998**
## Brief Information

This pull request is in the type of ([more info](https://github.com/angular/angular/blob/main/CONTRIBUTING.md#type) about types):

- [ ] build
- [ ] ci
- [ ] docs
- [ ] feat
- [ ] fix
- [ ] perf
- [ ] refactor
- [ ] test
- [x] style

Related issues ([all available keywords](https://docs.github.com/en/issues/tracking-your-work-with-issues/linking-a-pull-request-to-an-issue#linking-a-pull-request-to-an-issue-using-a-keyword)):

- resolve #495

## Details

Try my best to work out what a designer suggests in [this](https://www.figma.com/file/lHPwpACIjHsGQp1sgW4flf/Work-with-Frank?node-id=471%3A0) Figma:

image

## Checklist

- [x] I have read the [CONTRIBUTING](https://github.com/hypertrons/hypertrons-crx/blob/master/CONTRIBUTING.md) doc
- [x] I have signed the [CLA](https://cla-assistant.io/hypertrons/hypertrons-crx)

### CODE REVIEW

**Comment by tyn1998** [comment]
Now only the style of the Influence popup has been changed, but I can apply the changes to other popups soon. Take a look:

`Dark in English`:

image

`Light in Chinese`:

image

File: src/views/RepoHeaderLabelsView/InfluenceChart.tsx [hunk 1]
```diff
@@ -2,67 +2,72 @@ import React, { useEffect, useRef } from 'react';
...
     grid: {
-      top: '20%',
-      left: '12%',
-      width: '85%',
-      height: '60%',
+      top: '5%',
+      bottom: '5%',
+      left: '5%',
+      right: '5%',
+      containLabel: true,
```
Comment (tyn1998) [inline src/views/RepoHeaderLabelsView/InfluenceChart.tsx:49]: Thanks @pissang for pointing out the `containLabel` option to make the chart filling its parent container much more easier. This magic option saves my life and I think everyone who uses ECharts should know it :)

**Comment by tyn1998** [comment]
Changed `padding` to 8px instead of 16px the designer gives, since 8px makes the chart looks more similar to the one in Figma. And it makes the chart indeed looks better for me.

image

**Comment by tyn1998** [comment]
`Dark & English`、`Light & 中文`:

image

If someone can test this PR branch on his/her machine to confirm everything works well, then this PR can be merged with confidence. Who will?

**Comment by lhbvvvvv** [comment]
Hello! The bolding of the title is not obvious on my machine.
image

**Comment by Zzzzzhuzhiwei** [comment]
The bolding of the title text is also not obvious on my machine.
[8e8c2c5246e7e7a96db7b43f4206614](https://user-images.githubusercontent.com/115639837/199246018-5dde69d0-b512-4f66-a5de-1ab1f636bf39.jpg)
[e1b88fa76501a791ab49cc2cf2f443a](https://user-images.githubusercontent.com/115639837/199246149-dde88299-1ed3-47f5-8a91-73fdf2292848.jpg)

**Comment by tyn1998** [comment]
Thank you very much!

After trying, we figured out setting `font-weight` to `medium` or `500` not worked well on Windows. We'd better to use `bold` (600) since it works fine.

Though a little bold.

**Comment by tyn1998** [comment]
image

**Pull request merged**